pybossa/util.py

```diff
@@ -1010,22 +1010,29 @@ def get_user_pref_db_clause(user_pref, user_email=None):
     # expand user preferences as per sql format for jsonb datatype
     # single user preference with multiple value or
     # multiple user preferences with single/multiple values
-    _valid = ((k, v) for k, v in user_pref.items() if isinstance(v, list))
-    user_prefs = [{k: [item]} for k, pref_list in _valid
-                  for item in pref_list]
     assign_key = 'assign_user'
     location_key = 'locations'
     language_key = 'languages'
 
+    _valid = ((k, v) for k, v in user_pref.items() if isinstance(v, list))
+    turkey_spellings = {'Turkey', 'Türkiye', 'Turkiye'}
```

Comment:
This looks good for the one-time fix.

Let's also note, for future country name changes (2 or more), perhaps we could store this as a dictionary in the settings_local.py and then loop through the country name replacements accordingly.

For example:

```
ALTERNATE_COUNTRY_SPELLINGS = {
   "Turkey": ["Turkey", "Türkiye", "Turkiye"]
}
```

*To keep this PR simple, the existing change should be fine. Thank you.*